components/Navigation/DesktopNav/DesktopNav.module.scss

```diff
@@ -178,6 +186,29 @@
         right: -25px;
         top: -15px;
     }
+
+    &ExternalLinkIcon {
+        width: 12px;
+        height: 12px;
+        position: relative;
+        top: 2px;
+        right: -4px;
+        fill: var(--color--header-links);
+        transition: fill $transition-quick;
+        opacity: 0.8;
+
+        :global .themeDark :local & {
+            fill: var(--color--white);
+        }
+
+        .desktopNavItemLink:hover & {
+            fill: var(--color--coral);
+
+            :global .themeDark :local & {
+                fill: var(--color--white);
+            }
```

Comment:
Fix: Please set this variable to `--color--lagoon` to match the current site

<img width="1071" alt="image" src="https://github.com/torchbox/careers/assets/18164832/0592f6bf-e7f1-412f-964a-106904b1b40e">

Reply:
done, good spot